docs/api/Server.md

```diff
@@ -20,9 +20,10 @@ The `games` argument takes a list of game implementations
 An object that contains:
 
 1. run (_function_): A function to run the server.
-   Signature: (port, callback) => {}
+   Signature: (port, callback, apiPort) => {}
 2. app (_object_): The Koa app.
 3. db (_object_): The `db` implementation.
+4. singlePort (_boolean_): An option to run api and app using single port or two different ports. `false` by default.
```

Comment:
`singlePort` is part of the arguments to `Server()`.  This section documents the properties of the return value.

It's confusing because one of the lines above says "with the options shown below", but it is actually referring to the section below this.

Reply:
Fixed